File: wlxx-services/wlxx-system/wlxx-system-client/src/main/java/com/xzg/wlxx/system/client/entity/vo/OrgVo.java
package com.xzg.wlxx.system.client.entity.vo;

import com.xzg.wlxx.system.client.entity.po.OrgPo;
import lombok.Data;
import lombok.EqualsAndHashCode;

import java.util.List;

@Data
@EqualsAndHashCode(callSuper = true)
public class OrgVo extends OrgPo {

    private List<OrgVo> children;
}

File: wlxx-services/wlxx-system/wlxx-system-client/src/main/kotlin/com/xzg/wlxx/system/client/entity/vo/OrgVo.kt
package com.xzg.wlxx.system.client.entity.vo

import lombok.Data
import lombok.EqualsAndHashCode

@Data
@EqualsAndHashCode(callSuper = true)
data class OrgVo(
    var id: Long? = null,
    var orgCode: String? = null,

    var orgName: String? = null,
    var orgLevel: Int? = null,
    var orgId: Long? = null,
    var orgType: Int? = null,
    var children: List<OrgVo>? = null
)
